<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Geosaurus</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0">
    <widget class="QStackedWidget" name="stk_params">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_3">
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>362</width>
         <height>19</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Segments per 1/4 Circle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Mitre Limit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSlider" name="sld_distance">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>34</y>
         <width>362</width>
         <height>34</height>
        </rect>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>2</number>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>153</y>
         <width>362</width>
         <height>36</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QSlider" name="sld_segments">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>40</number>
          </property>
          <property name="pageStep">
           <number>5</number>
          </property>
          <property name="value">
           <number>8</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sld_mitre">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="pageStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="lbl_distance">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>13</y>
         <width>362</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Distance</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>74</y>
         <width>362</width>
         <height>19</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>End Cap</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Join</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>97</y>
         <width>362</width>
         <height>29</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="cmb_end_cap"/>
        </item>
        <item>
         <widget class="QComboBox" name="cmb_join"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sld_scale_x">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="pageStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>25</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sld_scale_y">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="pageStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>25</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="lbl_query">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Sample PostGIS Query</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lbl_postgis_version">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>PostGIS Version: 0.0.0 - Docs</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontal_layout">
     <item>
      <widget class="QComboBox" name="cmb_postgis_function">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="12" column="0">
    <widget class="QTextBrowser" name="txtb_query">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <spacer name="vertical_spacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="14" column="0">
    <widget class="QTextBrowser" name="txtb_result">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="lbl_result">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Sample Result</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lbl_postgis_function">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Choose PostGIS Function</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
